Probesys/bileto
# new: Change the status when answering to a ticket

Related to https://github.com/Probesys/bileto/issues/37

Changes proposed in this pull request:

- Render the full ticket page when posting a message
- Handle the scroll differently (`data-turbo-preserve-scroll` on success, `autofocus` on error)
- Add a status select box under the message to allow to change the ticket status in the same time

How to test the feature manually:

1. Create a ticket with a "new" status
2. Post an answer by selecting the "Pending" status
3. Check the status at the top of the page changes

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
